apps/dcos_dns/src/dcos_dns_key_mgr.erl

```diff
@@ -199,14 +187,16 @@ push_data_to_lashup(#{public := Pk, secret := Sk}) ->
     true.
 
 get_zookeepers() ->
+    ZooKeeperServers = application:get_env(dcos_dns, zookeeper_servers,
+                                           ?DEFAULT_ZOOKEEPER_SERVERS),
```

Comment:
nit: we'd prefer to avoid "standard" emacs indentation for erlang code. It would be just very painful to manage it and force that code style if there is someone who doesn't use such fancy plugins. We usually use 4 spaces everywhere with some exceptions, e.g. multiline list comprehensions.

Reply:
Styling: I would suggest to either keep all the arguments in one line (if possible) or move each argument in a separate line